providers/azurecompute-arm/src/main/java/org/jclouds/azurecompute/arm/domain/LoadBalancerSKU.java

```diff
@@ -0,0 +1,44 @@
+/*
```

Comment:
You probably can embed this type in the LoadBalancer type. It only makes sense inside the LB and naming is more meaningful as you just do `LoadBalancer.SKU`. Up to you.

Reply:
Same concern as before keeping the original MS names, but embedding it seems a good idea.